frontend/src/components/Board/Board.tsx

```diff
@@ -1,7 +1,339 @@
+  saved_date: new Date().toDateString(),
+  created_at: new Date().toDateString(),
+  user_id: '1111',
+  team_id: '2222',
+  name: 'FIT3162 Board',
+  description:
+    'This board is used to monitor the progress of the FIT3162 CS_20 Project.',
+};
+
+// list table
+const columns: IColumn[] = [
+  {
+    id: '1',
+    board_id: '123456789',
+    board_version: 1,
+    index: 1,
+    name: 'Backlog',
+    created_at: new Date().toDateString(),
+    cards: [
+      {
+        id: '1',
+        list_id: '1',
+        index: 1,
+        user_creator: '1',
+        user_assigned: undefined,
+        title: 'Design the UI!',
+        description:
+          'Create a figma design of the initial UI to demo to the clients.',
+        deadline: undefined,
+        created_at: new Date().toDateString(),
+      },
+    ],
+  },
+  {
+    id: '2',
+    board_id: '123456789',
+    board_version: 1,
+    index: 2,
+    name: 'Doing',
+    cards: [
+      {
+        id: '2',
+        list_id: '2',
+        index: 1,
+        user_creator: '1',
+        user_assigned: '1',
+        title: 'Get to know the team!',
+        description: 'Do ice breaker activities to understand the team better.',
+        deadline: new Date().toDateString(),
+        created_at: new Date().toDateString(),
+      },
+    ],
+    created_at: new Date().toDateString(),
+  },
+  {
+    id: '3',
+    board_id: '123456789',
+    board_version: 1,
+    index: 3,
+    name: 'Code Review',
+    cards: [],
+    created_at: new Date().toDateString(),
+  },
+  {
+    id: '4',
+    board_id: '123456789',
+    board_version: 1,
+    index: 4,
+    name: 'Testing',
+    cards: [],
+    created_at: new Date().toDateString(),
+  },
+  {
+    id: '5',
+    board_id: '123456789',
+    board_version: 1,
+    index: 5,
+    name: 'Done',
+    cards: [
+      {
+        id: '3',
+        list_id: '5',
+        index: 1,
+        user_creator: '1',
+        user_assigned: '1',
+        title: 'Watch Seminar Recordings',
+        description:
+          'Watch the Seminar recordings before meeting with the team.',
+        deadline: new Date().toDateString(),
+        created_at: new Date().toDateString(),
+      },
+    ],
+    created_at: new Date().toDateString(),
+  },
+];
+
+// const cards = [
+//   {
+//     id: "1",
+//     list_id: "1",
+//     index: 1,
+//     user_creator: "1",
+//     user_assigned: undefined,
+//     title: "Design the UI!",
+//     description:
+//       "Create a figma design of the initial UI to demo to the clients.",
+//     deadline: undefined,
+//     created_at: new Date().toDateString(),
+//   },
+//   {
+//     id: "2",
+//     list_id: "2",
+//     index: 2,
+//     user_creator: "1",
+//     user_assigned: "1",
+//     title: "Get to know the team!",
+//     description: "Do ice breaker activities to understand the team better.",
+//     deadline: new Date().toDateString(),
+//     created_at: new Date().toDateString(),
+//   },
+// ];
+
+const team = {
```

Comment:
Are `team` and `members` gonna be used?

Reply:
yea, to show who else has access to the board and assigning cards to people